src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Models/Capabilities.cs

```diff
@@ -0,0 +1,89 @@
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+
+namespace Altinn.Common.EFormidlingClient.Models
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Capabilities"/> class.
+    /// </summary>
+    public class Capabilities
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Capabilities"/> class.
+        /// </summary>
+        /// <param name="capabilities">Capabilities</param>
+        public Capabilities(List<Capability> capabilities)
+        {
+            Capability = capabilities;
+        }
+
+        /// <summary>
+        ///  Gets or sets the Capabilities
+        /// </summary>
+        [JsonPropertyName("capabilities")]
+        public List<Capability> Capability { get; set; }
```

Comment:
We usually define the propertie above the constructor.
Could the class inherit from one of the collection interfaces rather than defining a sub-property that doesn't add much value ?